controller/uninstall_controller.go

```diff
@@ -319,6 +325,13 @@ func (c *UninstallController) deleteLease() error {
 }
 
 func (c *UninstallController) deleteCRDs() (bool, error) {
+	if shareManagers, err := c.ds.ListShareManagers(); err != nil {
```

Comment:
Previously this was at the bottom of this function, and should be moved back there. 
Since it's preferable to cleanup the volumes first. otherwise the volume controller will try to recreate a share-manager.

Reply:
A good suggestion! @cclhsu please help improve this part. Thanks!